syntax = "proto3";

message HelloText{
    string hello = 1;
}
message StringArray{
    repeated string data = 1;
}
message Void{}
service EremApi{
    rpc getHelloMsg(Void) returns (HelloText){}
    rpc getArray(Void) returns (StringArray){}
}